frontend/public/locales/fr/translation.json

```diff
@@ -856,8 +856,8 @@
   "Create wizard steps": "Créer des étapes d’assistant",
   "Create, update, scale, and remove clusters reliably, consistently using an open source programming model that supports and encourages Infrastructure as Code best practices and design principles.": "Créez, mettez à jour, mettez à l’échelle et supprimez des clusters de manière fiable et cohérente à l’aide d’un modèle de programmation open source qui prend en charge et encourage les meilleures pratiques et les principes de conception d’infrastructure en tant que code (Infrastructure as Code).",
   "createCluster.card.noHosts.alert": "Pour utiliser {{ controlPlaneTypeTxt }}, accédez à <a>Inventaire d’hôtes</a> et ajoutez des hôtes.<br />1. Pour utiliser les environnements d’infrastructure existants, assurez-vous que des hôtes sont disponibles ou ajoutez des hôtes.<br />2. Pour utiliser un nouvel environnement d’infrastructure, créez-en un, puis ajoutez-y des hôtes.",
-  "createCluster.card.noInfraEnvs.alert": "Pour utiliser le plan de contrôle hébergé, créez un environnement d’infrastructure dans l’ <a>inventaire de l’hôte</a>.",
-  "createCluster.card.noInfraEnvsOrHosts.alert": "Pour utiliser des hôtes existants, créez un environnement d’infrastructure et ajoutez des hôtes dans l’ <a>inventaire des hôtes</a>.",
+  "createCluster.card.noInfraEnvs.alert": "Pour utiliser le plan de contrôle hébergé, créez un environnement d’infrastructure dans l’<a>Inventaire d’hôtes</a>.",
+  "createCluster.card.noInfraEnvsOrHosts.alert": "Pour utiliser des hôtes existants, créez un environnement d’infrastructure et ajoutez des hôtes dans l’<a>Inventaire d’hôtes</a>.",
```

Comment:
The text in the `<a>` tags refers to a navigation menu item that is translated. So I was not flagging a singular/plural issue here, but that the text should match exactly what appears in the UI.